src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -37,6 +43,8 @@ public class WasmAppBuilder : WasmAppBuilderBaseTask
     // </summary>
     public ITaskItem[]? ExtraConfig { get; set; }
 
+    private static SHA256 HashAlgorithm = SHA256.Create();
```

Comment:
Instances of `SHA256` are not thread-safe. If two different instances of `WasmAppBuilder` are sharing this instance and it gets used concurrently, it can lead to incorrect hashes or exceptions.